packages/terra-form-select/src/shared/_OptGroup.jsx

```diff
@@ -39,7 +39,7 @@ const OptGroup = ({
 }) => {
   const theme = React.useContext(ThemeContext);
   return (
-    <li className={cx('opt-group', { 'is-disabled': disabled }, theme.className)} role="group">
+    <li className={cx('opt-group', { 'is-disabled': disabled }, theme.className)} role="option" aria-selected="false">
```

Comment:
This is one I'm not too sure about but documentation on the right thing to do here is tricky to find.